Motor reduction

josephhadad05 opened the pull request.

josephhadad05 committed:
Changed motors' ids

josephhadad05 committed:
Changed class Climber to function with one motor

josephhadad05 committed:
Changed class Tilt to function with one motor

josephhadad05 committed:
Changed commands to function with one motor

josephhadad05 committed:
Got rid off Hook class

josephhadad05 reviewed (commented):
Looks good, constants for tilt do need to be recalculated though. Mainly gear ratio, static voltage, backlash, and pid.

AndreyGL0290 commented:
Hey, what about the conversion factor? Because I assume that the gear ratio changed on that one motor that we left

josephhadad05 commented:
Currently running a 27:1 ratio. Units should be in radians.

josephhadad05 committed:
Gear ratio fix
Changed from 125:1 to 1:27
Current limit increased from 10 to 20
Adjusted PID values
